Src/AutoFixtureUnitTest/DataAnnotations/RangeAttributeRelayTest.cs

```diff
@@ -125,16 +124,12 @@ public void CreateWithRangeAttributeRequestReturnsCorrectResult(Type type, objec
         [InlineData("Property", "-2.2", "-1.1")]
         [InlineData("Property", 10.0, 20.0)]
         [InlineData("Property", -2.0, -1.0)]
-        [InlineData("Property", 10, 20)]

```

Comment:
Why are those removed?

Reply:
If you look around you'll notice that are duplicated 😉 You are the author of these lines so it seems that you mistakenly introduced that by copy-paste 😄

Reply:
![image](https://user-images.githubusercontent.com/1074182/29965260-9e16a2a2-8f15-11e7-89aa-27158b26dac5.png)

Reply:
👍 👍 😄